csharp/ql/lib/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -1149,7 +1149,7 @@ private module Cached {
     } or
     TCapturedVariableContent(VariableCapture::CapturedVariable v) or
     TDelegateCallArgumentContent(int i) {
-      i = [0 .. max(int j | j = any(DelegateCall dc).getNumberOfArguments())]
+      i = [0 .. max(any(DelegateCall dc).getNumberOfArguments())]
```

Comment:
Still missing the `- 1`.

Reply:
Ah yes, you are right! Thank you for paying close attention!

Reply:
I suppose we also want to consider DelegateLikeCall instead.

Reply:
Oh, yes.